packages/utils/upgrade/src/modules/project/__tests__/project.test.ts

```diff
@@ -108,21 +108,21 @@ describe('Project', () => {
     test('Succeed for valid AppProject', () => {
       vol.fromNestedJSON(appVolume, defaultCWD);
 
-      const project = projectFactory(defaultCWD);
+      const project = projectFactory(defaultCWD) as AppProject;
```

Comment:
We could have a small assertIsAppProject that does the casting + fails the test if it's not an app project